h3-widgets/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>CARTO + deck.gl - H3 Spatial Index Widgets</title>
+  </head>
+  <body>
+    <div id="map"></div>
+    <canvas id="deck-canvas"></canvas>
+    <div id="top-left">
+      <div id="story-card">
+        <p class="overline">✨👀 You're viewing</p>
+        <h2>CARTO Widgets for H3 Spatial Index</h2>
+        <p>
+          More info about
+          <a
+            href=" https://carto.com/blog/carto-spatial-features-urbanity-climatology-elevation"
+            rel="noopener noreferrer"
+            target="_blank"
+            >H3 Spatial Index</a
+          >
+        </p>
+  
+        <div class="layer-controls">
+          <p class="overline">Variable</p>
+          <select name="variable" id="variable" class="select"></select>
+          <p class="overline">Agreggation method applied</p>
+          <p><pre id="agg-method"></pre></p>
```

Comment:
I think Prettier should flag this one too but just a heads up that the `<pre/>` cannot be inside a `<p/>`, maybe `<code />` instead.

Reply:
Well this was copied from the previous examples, but we can change it to `code` or just remove this section

Reply:
Ah ok! I'll defer to @srtena if he prefers something else, but your changes look good to me.

Reply:
Agree it should be `code`... Those two old examples have a few other issues 🙃 I'll try to find some time to improve them but low priority